packages/server/locales/en.json

```diff
@@ -689,4 +689,15 @@
 	"There must be a value in every row": "There must be a value in every row",
 	"Different rows must have different values for this field": "Different rows must have different values for this field",
 	"Add more fields to enable edit": "Add more fields to enable edit"
```

Comment:
need a comma at the end of this line; everything else looks good

Reply:
Corrected according comment of @glutamate